Bench-Kitura-Blog/build.sh

```diff
@@ -31,13 +31,32 @@ Darwin)
   ;;
 esac
 
-# Build 'new' version
+# Build 'new' and "nio" version
 # If $REPO and $NEW_COMMIT are set, edit package $REPO to $NEW_COMMIT before building
+
+function cloneProject {
+  local projectPath=$1
+  local newPath=$2
+  pushd $projectPath
+  swift package reset
+  popd
+  if [ -d $newPath ]; then
+    preserveDir $newPath
+  fi
+  cp -R -p $projectPath $newPath
+}
+
 pushd $dir/latest/
 build "$dir/newBuild" "$REPO" "$NEW_COMMIT"
 popd
 
+cloneProject $dir/latest $dir/latest-nio
```

Comment:
You need to do this _before_ building latest because the first thing `cloneProject` does is go into latest and do a `swift package reset`.